openwisp_utils/admin_theme/static/admin/css/openwisp.css

```
@@ -877,7 +881,7 @@ input[type="button"]:hover,
 .title-wrapper h2::after {
   content: ")";
 }
-#content.title-wrapper .object-tools {
+#content .title-wrapper .object-tools {
```

Was this changed in Django or was it wrong here in our code in the first place?

If Django changed this, then I think we must keep backward compatibility with older versions, eg:
```suggestion
#content.title-wrapper .object-tools,
#content .title-wrapper .object-tools {
```

---

I updated this class in https://github.com/openwisp/openwisp-utils/pull/462 to override the Django defaults. There was a typo in the PR.